cloud/blockstore/libs/storage/partition/part_actor_writemergedblocks.cpp

```diff
@@ -528,7 +559,7 @@ void TPartitionActor::WriteMergedBlocks(
         requestInBuffer.Data.ReplyLocal,
         shouldAddUnconfirmedBlobs,
         std::move(requestInBuffer.Data.Handler),
-        checksumsEnabled
+        checksumsEnabled ? State->GetBlockSize() : 0
```

Comment:
Maybe annotate this argument, for better reading?

Reply:
I think that here the meaning of this argument is obvious